usermembership/tool/src/java/org/sakaiproject/umem/tool/ui/UserListBean.java

```diff
@@ -342,8 +351,8 @@ private void doSearch() {
 								u.getEmail(), 
 								u.getType(), 
 								msgs.getString("user_auth_internal"), 
-								(u.getCreatedTime() == null) ? "" : u.getCreatedTime().toStringLocalDate(), 
-								(u.getModifiedTime() == null) ? "" : u.getModifiedTime().toStringLocalDate()
+								u.getCreatedDate(), 
```

Comment:
These blow up on external users that don't have a created date set. There's a issue out to have the sample provided users included in searches.

https://jira.sakaiproject.org/browse/SAK-33647

Reply:
https://github.com/sakaiproject/sakai/pull/5045